src/Xamarin.Android.Build.Utilities/Sdks/MonoDroidSdkBase.cs

```diff
@@ -34,13 +36,15 @@ abstract class MonoDroidSdkBase
 		// runtimePath: contains Mono.Android.DebugRuntime-*.apk
 		// binPath:     contains mandroid
 		// mscorlibDir: contains mscorlib.dll
-		public void Initialize (string runtimePath = null, string binPath = null, string bclPath = null)
+		public void Initialize (string expectedRuntimePath = null, string binPath = null, string bclPath = null)
```

Comment:
It is not just renaming, it is preserving the actual value that was passed to the method to report later as a warning.